clang/lib/Driver/OffloadBundler.cpp

```diff
@@ -1365,6 +1367,20 @@ Error OffloadBundler::BundleFiles() {
   return Error::success();
 }
 
+bool CheckTripleIsExcluded(
+    StringRef Triple, const std::vector<std::string> &ExcludedTargetNames) {
+  // NOTE: "-sycldevice" Triple component has been deprecated.
```

Comment:
Does this consumption need to happen at a higher level and/or in other places? Not a PR blocker?

Thanks

Reply:
As far as I can see everything seems to work without supporting in other places.
The appropriate way could be to ask fpga folks to rebuild their libraries so that this Triple component disappeared from the input. Then we could remove corresponding support.